duffy/worker.py

```diff
@@ -0,0 +1,94 @@
+'''
+<TODO>
+This is a [WIP] python file to update worker from v1.
+'''
+
+from config import ProdConfig
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import json
+import subprocess
+
+import beanstalkc
+bs_obj = beanstalkc.Connection(host='127.0.0.1', parse_yaml=False)
+bs_obj.watch('requests')
+bs_obj.ignore('default')
+
+app = Flask(__name__)
+app.config['DEBUG'] = ProdConfig.DEBUG
+app.config['DEBUG_TB_ENABLED'] = ProdConfig.DEBUG_TB_ENABLED
+app.config['ENV'] = ProdConfig.ENV
+app.config['SQLALCHEMY_DATABASE_URI'] = ProdConfig.SQLALCHEMY_DATABASE_URI
+
+db_obj = SQLAlchemy(app)
+
+def connection_check():
+    '''
+    This function checks and establishes sqllite Connection.
+    '''
+    try:
+        dummy_data = stock.query.first()
+    except:
+        global db_obj
+        global app
+        db_obj = SQLAlchemy(app)
+
+
+def provision(json_jobs):
+    '''
+    This function executes ansible playbook local-ci-deploy.yml && updates db
+    accordingly
+    '''
+    cmd_line = "date ; cd /srv/code/ansible  ; /usr/bin/ansible-playbook playbooks/local-ci-deploy.yml --limit %s.ci.centos.org --extra-vars 'centos_dist=%s centos_arch=%s'" % (json_jobs['hostname'], json_jobs['ver'], json_jobs['arch'])
+    return_code = subprocess.call(cmd_line, shell=True)
+    print(cmd_line)
+    print('Returned : ', return_code)
+    hostname_var = json_jobs['hostname']
+    connection_check()
+    session = stock.query.filter_by(hostname=hostname_var)
```

Comment:
Instead of doing `stock.query` you'll want to import the Host model from models.py (`from duffy.models.nodes import Host` at the top)

That way you can do either db.query(Host)..., or Host.query...